src/librustc_unicode/u_str.rs

```diff
@@ -369,7 +369,7 @@ impl<'a> DoubleEndedIterator for Graphemes<'a> {
 }
 
 // https://tools.ietf.org/html/rfc3629
-static UTF8_CHAR_WIDTH: [u8; 256] = [
+const UTF8_CHAR_WIDTH: [u8; 256] = [
```

Comment:
Constants can indirectly be required to go into the metadata, for example if they're used in an `#[inline]` function. If this function is never inlined then I don't believe there's a benefit one way or the other for using a `static` or a `const`, so it seems more appropriate to use a `static` as it's an assertion that this should *never* go into the metadata.